bokeh/resources.py

```diff
@@ -33,25 +33,36 @@ def _cdn_base_url():
 def _get_cdn_urls(version=None, minified=True):
     if version is None:
         version = __version__.split('-')[0]
-    min = ".min" if minified else ""
+
+    # check the 'dev' fingerprint
+    dev = version.split('.')[-2]
+    # check if we want minified js and css
+    _min = ".min" if minified else ""
```

Comment:
OK, no that's fine I just thought we were only uploading minified for some reason. If we are uploading both then keep this as-is.